### DISCUSSION TITLE
Max Queries?

### DISCUSSION THREAD

**Discussion by Pogchampion69** [omkarcloud/google-maps-scraper#35] [Q&A]
Curious on max amount of keywords and also the format for adding filters on multiple keywords do I need to include the filters within each query or will it default to all of them? Also just for reference when it comes to coding I may as well be brain dead trying to learn python rn.

**Comment by Chetan11-dev** [comment]
You can put as many keyword as needed. Also you need to include filters in each query like
```
queries = [
    {
        "keyword": "restaurants in delhi",
        "min_reviews": 5 ,
        "max_reviews": 100,
        "has_phone": True,

    },
    {
        "keyword": "restaurants in bangalore",
        "min_reviews": 5 ,
        "max_reviews": 100,
        "has_phone": True,

    }
]

```
